tests/data_dir/phylo_beta_diversity/valid/small_test/pam.csv

```diff
@@ -1,4 +1,4 @@
-, sp1, sp2, sp3, sp4, sp5, sp6
+,sp1,sp2,sp3,sp4,sp5,sp6
```

Comment:
Changing this shouldn't cause any problems, but the code to read the data should handle this scenario.  Can you tell me where exactly this was failing to load so I can address it?